content/applications/services/helpdesk.rst

```diff
@@ -238,6 +238,23 @@ Select a stage to fold temporarily, then click the :guilabel:`⚙️ (gear)` ico
    Manually folding a stage from the Kanban view is temporary and does **not** close the tickets in
    the stage.
 
+Merge tickets
+=============
+
+If duplicate tickets are found in *Helpdesk*, they can be combined into a single ticket using the
+*merge* feature.
+
+.. important::
+   The *merge* feature is **only** accessible if the :doc:`Data Cleaning
+   <../productivity/data_cleaning>` application is installed on the database.
+
+To merge two or more tickets, navigate to :menuselection:`Helpdesk app --> Tickets --> All Tickets`.
+Identify the tickets to be merged and tick the checkbox at the far left of each ticket. Then, click
+the :icon:`fa-cog` :guilabel:`Actions` icon, and select :guilabel:`Merge` from the drop-down menu.
+Doing so opens a new page where the selected tickets are listed with their :guilabel:`Similarity`
+rating. From here, tickets can be :ref:`merged <data_cleaning/merge-records>` or
+:guilabel:`Discarded`, as desired.
+
```

Comment:
Just a suggestion- I noticed there's a 'Differences' field you can make visible on the list. Might be nice to mention so they can take a look at the differences to see if they really want to combine them or not.

Reply:
I didn't want to duplicate the information from the data cleaning doc, so I didn't want to go too far into that information, this was just to add the fact that the feature is accessible from helpdesk.